### PR TITLE
Fixes #7 - setting max force

### LINKED ISSUE #7: Bug in HandCommander.set_max_force
The expected and documented behaviour

```
hand_cmd.set_max_force('rh_FFJ3', 200)
```

does not work. Right now what works is:

```
hand_cmd.set_max_force('FFJ3', 200)
```

Modify the function to strip the prefix from the joint_name argument.

### PR DESCRIPTION
**Pull request by ugocupcic**
Needs [that one](https://github.com/shadow-robot/sr_interface/pull/9) to be merged first

### CODE REVIEW

File: sr_hand_kinematics/hand_kinematics/test/test_kinematics_as_plugin.cpp [hunk 1]
```diff
@@ -76,7 +76,7 @@ class MyTest
     {
       kinematics_solver_ = kinematics_loader_->createInstance(plugin_name);
     }
-    catch (pluginlib::PluginlibException &ex  // handle the class failing to load
+    catch (pluginlib::PluginlibException &ex)  // handle the class failing to load
```
Comment (ugocupcic) [inline sr_hand_kinematics/hand_kinematics/test/test_kinematics_as_plugin.cpp:79]: not related but it fixes the tests on my machine

File: sr_multi_moveit/sr_multi_moveit_test/test/test_planners_left_arm.test [hunk 2]
```diff
@@ -1,25 +1,25 @@
-<launch >
-    <include file = "$(find left_sr_ur10_moveit_config)/launch/demo.launch" >
-        <arg name = "visualization" value = "false" / >
-    < / include >
+<launch>
```
Comment (ugocupcic) [inline sr_multi_moveit/sr_multi_moveit_test/test/test_planners_left_arm.test:1]: same here

File: sr_multi_moveit/sr_multi_moveit_test/test/test_planners_right_arm.test [hunk 3]
```diff
@@ -1,25 +1,25 @@
-<launch >
-    <include file = "$(find right_sr_ur10_moveit_config)/launch/demo.launch" >
-        <arg name = "visualization" value = "false" / >
-    < / include >
+<launch>
```
Comment (ugocupcic) [inline sr_multi_moveit/sr_multi_moveit_test/test/test_planners_right_arm.test:1]: same here

File: sr_robot_commander/test/test_hand_commander.test [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<launch>
+    <include file="$(find right_sr_ur10_moveit_config)/launch/demo.launch">
+        <arg name="visualization" value="false"/>
+    </include>
+
+    <test test-name="test_hand_commander" pkg="sr_robot_commander" type="test_hand_commander.py"
+          time-limit="300.0" retry="1"/>
+</launch>
```
Comment (VahidAminZ) [inline sr_robot_commander/test/test_hand_commander.test]: Needs a new line at the end of file.

File: sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py [hunk 5]
```diff
@@ -106,3 +107,14 @@ def get_tactile_state(self):
...
+    def _strip_prefix(self, joint_name):
+        """
+        Strips the prefix from the joint name (e.g. rh_ffj3 -> ffj3) if present, returns the joint name otherwise.
+
+        We know that all joint names for the shadow hands are 4 char long. So we only keep the last 4 chars.
+
+        @param joint_name the joint name
+        @return stripped joint name
+        """
+        return joint_name[4:]
```
Comment (VahidAminZ) [inline sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: shouldn't it be joint_name[3:] or joint_name[-4:]?

File: sr_hand_kinematics/hand_kinematics/test/test_kinematics_as_plugin.cpp (hunk 1 above)
Comment (VahidAminZ) [inline sr_hand_kinematics/hand_kinematics/test/test_kinematics_as_plugin.cpp:79]: how was it working before?

Comment (ugocupcic) [inline sr_hand_kinematics/hand_kinematics/test/test_kinematics_as_plugin.cpp:79]: it wasn't :)

File: sr_robot_commander/test/test_hand_commander.test (hunk 4 above)
Comment (ugocupcic) [inline sr_robot_commander/test/test_hand_commander.test]: fixed

**Pull request merged**